inc/admin/upgrader.php

```diff
@@ -357,6 +357,13 @@ function imagify_reset_opcache() {
 			return;
 		}
 
+		$opcache_enabled = ini_get( 'opcache.enable' );
```

Comment:
Add filter_var() and simplify the bailout condition:
```suggestion
		$opcache_enabled = filter_var( ini_get( 'opcache.enable' ), FILTER_VALIDATE_BOOLEAN );
		
		if ( ! $opcache_enabled ) {
			$can_reset = false;
			return;
		}
```

Reply:
Nice, I didn't know this filter could be used that way.